### PR TITLE
feature: data process for SMPL & Rendering

### PR DESCRIPTION
**Pull request by Leoooo333**

### CODE REVIEW

File: docs/data_processors/README.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
+# 💃 SMPL & Rendering
+
+Try Champ with your dance videos! It may take time to setup the environment, follow the instruction step by step🐢, report issue when necessary.
+
+
```
Comment (AricGamma) [inline docs/data_processors/README.md:5]: Please supplement test environment.

File: README.md [hunk 2]
```diff
@@ -26,15 +26,32 @@
 
 https://github.com/fudan-generative-vision/champ/assets/82803297/b4571be6-dfb0-4926-8440-3db229ebd4aa
 
-# Framework
+# 🏆 Framework
```
Comment (siyuzhu-fudan) [inline README.md:29]: -remove

File: README.md [hunk 3]
```diff
@@ -54,14 +71,13 @@ Create conda environment:
 poetry install --no-root
 ```
 
-# Download pretrained models
+# 💾 Download pretrained models
```
Comment (siyuzhu-fudan) [inline README.md:74]: -remove

File: README.md [hunk 4]
```diff
@@ -54,14 +71,13 @@ Create conda environment:
...
 
 1. Download pretrained weight of base models:
 
    - [StableDiffusion V1.5](https://huggingface.co/runwayml/stable-diffusion-v1-5)
    - [sd-vae-ft-mse](https://huggingface.co/stabilityai/sd-vae-ft-mse)
    - [image_encoder](https://huggingface.co/lambdalabs/sd-image-variations-diffusers/tree/main/image_encoder)
-
```
Comment (siyuzhu-fudan) [inline README.md:64]: -remove

File: README.md [hunk 5]
```diff
@@ -95,7 +111,7 @@ Finally, these pretrained models should be organized as follows:
     `-- v1-inference.yaml
 ```
 
-# Inference
+# 🐳 Inference
```
Comment (siyuzhu-fudan) [inline README.md:114]: -remove

File: README.md [hunk 6]
```diff
@@ -116,19 +132,18 @@ You can also extract the driving motion from any videos and then render with Ble
...
 
-Champ ComfyUI tutorial see [here](https://www.youtube.com/watch?app=desktop&v=cbElsTBv2-A)!
 
-# Acknowledgements
+# 👏 Acknowledgements
 
-We thank the authors of [MagicAnimate](https://github.com/magic-research/magic-animate), [Animate Anyone](https://github.com/HumanAIGC/AnimateAnyone), and [AnimateDiff](https://github.com/guoyww/AnimateDiff) for their excellent work. Our project is built upon [Moore-AnimateAnyone](https://github.com/MooreThreads/Moore-AnimateAnyone), and we are grateful for their open-source contributions.
+We thank the authors of [MagicAnimate](https://github.com/magic-research/magic-animate), [Animate Anyone](https://github.com/HumanAIGC/AnimateAnyone), and [AnimateDiff](https://github.com/guoyww/AnimateDiff) for their excellent work. Our project is built upon [Moore-AnimateAnyone](https://github.com/MooreThreads/Moore-AnimateAnyone), [4D-Humans](https://github.com/shubham-goel/4D-Humans), [DWPose](https://github.com/IDEA-Research/DWPose) and we are grateful for their open-source contributions.
```
Comment (siyuzhu-fudan) [inline README.md:142]: -remove

File: README.md [hunk 7]
```diff
@@ -116,19 +132,18 @@ You can also extract the driving motion from any videos and then render with Ble
...
 
-# Roadmap
 
-Visit [our roadmap](https://github.com/fudan-generative-vision/champ/blob/master/docs/ROADMAP.md) to preview the future of Champ.
 
-# Citation
+# 🌟 Citation
```
Comment (siyuzhu-fudan) [inline README.md:146]: remove

File: README.md [hunk 8]
```diff
@@ -143,7 +158,7 @@ If you find our work useful for your research, please consider citing the paper:
 }
 ```
 
-# Opportunities available
+# 👋 Opportunities available
```
Comment (siyuzhu-fudan) [inline README.md:161]: remove

File: README.md [hunk 9]
```diff
@@ -26,15 +26,32 @@
...
+
+- **`2024/04/08`**: ✨✨✨SMPL & Rendering scripts released! Champ your dance videos now.💃🤸‍♂️🕺
+  
+- **`2024/03/30`**: 🚀🚀🚀Watch this amazing [video tutorial](https://www.youtube.com/watch?app=desktop&v=cbElsTBv2-A). It's based on the **unofficial**(unstable) [Champ ComfyUI](https://github.com/kijai/ComfyUI-champWrapper?tab=readme-ov-file)🥳.
+  
+- **`2024/03/27`**: Cool Demo on [replicate](https://replicate.com/camenduru/champ)🌟, Thanks [camenduru](https://github.com/camenduru)!👏
+
+- **`2024/03/27`**: Visit our [roadmap🕒](https://github.com/fudan-generative-vision/champ/blob/master/docs/ROADMAP.md) to preview the future of Champ.
+
+# 🐟 Installation
```
Comment (siyuzhu-fudan) [inline README.md:44]: donot introduce the news in this merge request.

File: README.md [hunk 10]
```diff
@@ -26,15 +26,32 @@
...
 
 - System requirement: Ubuntu20.04/Windows 11, Cuda 12.1
 - Tested GPUs: A100, RTX3090
 
+Git clone Champ with following command:
+```bash
+  git clone https://github.com/fudan-generative-vision/champ
+```
+
+
```
Comment (siyuzhu-fudan) [inline README.md:54]: donot update the readme.md irrelevant to smpl & rendering.

File: requirements.txt [hunk 11]
```diff
@@ -1,14 +1,23 @@
 accelerate==0.21.0
+controlnet-aux==0.0.7
```
Comment (siyuzhu-fudan) [inline requirements.txt:2]: relevant to smpl & rendering?

File: requirements.txt [hunk 12]
```diff
@@ -1,14 +1,23 @@
...
 einops==0.4.1
 imageio==2.33.0
 imageio[ffmpeg]
+matplotlib==3.8.3
 numpy==1.23.5
 omegaconf==2.2.3
 Pillow==9.5.0
 safetensors==0.4.2
+onnxruntime-gpu==1.13.1
+open-clip-torch==2.20.0
```
Comment (siyuzhu-fudan) [inline requirements.txt:13]: relevant to smpl & rendering?

File: requirements.txt (hunk 11 above)
Comment (Leoooo333) [reply requirements.txt:2]: Necessary for DWPose

**Review by siyuzhu-fudan — APPROVED** [review]

**Pull request merged**